packages/ckeditor5-dev-transifex/lib/createpotfiles.js

```diff
@@ -3,16 +3,15 @@
  * For licensing, see LICENSE.md.
  */
 
-'use strict';
-
-const path = require( 'path' );
-const fs = require( 'fs-extra' );
-const del = require( 'del' );
-const defaultLogger = require( '@ckeditor/ckeditor5-dev-utils' ).logger();
-const { findMessages } = require( '@ckeditor/ckeditor5-dev-translations' );
-const { verifyProperties } = require( './utils' );
-
-const langContextSuffix = path.join( 'lang', 'contexts.json' );
+import path from 'path';
+import fs from 'fs-extra';
+import del from 'del';
+import { logger as utilsLogger } from '@ckeditor/ckeditor5-dev-utils';
+import { findMessages } from '@ckeditor/ckeditor5-dev-translations';
+import { verifyProperties } from './utils.js';
+
+let defaultLogger;
```

Comment:
Could it be defined inside the `createPotFiles()` function and passed down whenever we need it?

Reply:
Review all files you changed, as you often used this pattern.